server/views/partials/header.njk

```diff
@@ -36,7 +36,7 @@
           </li>
 
           <li class="hmpps-header__navigation__item">
-            <a data-qa="signOut" class="hmpps-header__link" href="/link/request-link?force=true">Sign out</a>
+            <a data-qa="signOut" class="hmpps-header__link" href="/legal-sender/sign-out">Sign out</a>
```

Comment:
Use the new route to sign out, which in turn redirects to the correct sign out route